test-scripts/tomcat-deployment/deploy.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# This script is responsible for deploying the Tomcat server on AWS instances
+
+# For the deployment script to run infra_eps and key.pem files should exist
+
+infraEPFileName=infra_eps
+outputFileName=deployment_eps
+
+#Extracting the IP of the tomcat server
+endpoint=$(cat $infraEPFileName)
```

Comment:
This logic may fail when we add a new property to the infra provision output. We should rather parse the properties file and get the `serverHost` property.

This gist show how to parse properties files: https://gist.github.com/marcelbirkner/9b133f800d7d3fc5d828